_analyzers/token-filters/asciifolding.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: ASCIIFolding
+parent: Token filters
+nav_order: 20
+---
+
+# ASCIIFolding token filter
+
+`asciifolding` is a token filter that converts non-ASCII characters into their closest ASCII equivalents. For example *é* becomes *e*, *ü* becomes *u* and *ñ* becomes *n*. This process is also known as *transliteration*.
+
+
+`asciifolding` token filter offers a number of benefits:
+
+  - __Enhanced Search Flexibility__: Users often omit accents or special characters when typing queries. ASCIIFolding ensures that such queries still return relevant results.
```

Comment:
`**` instead of `__` so Vale doesn't choke:
```suggestion
  - **Enhanced search flexibility**: Users often omit accents or special characters when typing queries. The `asciifolding` token filter ensures that such queries still return relevant results.
```